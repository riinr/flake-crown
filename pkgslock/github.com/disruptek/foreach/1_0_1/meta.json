{
  "name": "foreach",
  "version": "1.0.1",
  "author": "disruptek",
  "desc": "A sugary for loop with syntax for typechecking loop variables",
  "license": "MIT",
  "skipDirs": [],
  "skipFiles": [],
  "skipExt": [],
  "installDirs": [],
  "installFiles": [],
  "installExt": [],
  "requires": [
    {
      "name": "nim",
      "str": ">= 0.20.0",
      "ver": {
        "kind": "verEqLater",
        "ver": "0.20.0"
      }
    }
  ],
  "bin": [],
  "binDir": "",
  "srcDir": "",
  "backend": "c",
  "ref": "refs/tags/1.0.1"
}