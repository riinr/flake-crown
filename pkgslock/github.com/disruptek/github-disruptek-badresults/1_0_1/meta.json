{
  "name": "badresults",
  "version": "1.0.1",
  "author": "disruptek",
  "desc": "a less fascist fork of nim-result",
  "license": "MIT",
  "skipDirs": [],
  "skipFiles": [],
  "skipExt": [],
  "installDirs": [],
  "installFiles": [],
  "installExt": [],
  "requires": [
    {
      "name": "nim",
      "str": ">= 1.0.6",
      "ver": {
        "kind": "verEqLater",
        "ver": "1.0.6"
      }
    }
  ],
  "bin": [],
  "binDir": "",
  "srcDir": "",
  "backend": "c",
  "ref": "refs/tags/1.0.1"
}