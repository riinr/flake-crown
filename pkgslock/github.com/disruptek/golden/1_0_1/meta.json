{
  "name": "golden",
  "version": "1.0.1",
  "author": "disruptek",
  "desc": "a nim benchmark",
  "license": "MIT",
  "skipDirs": [],
  "skipFiles": [],
  "skipExt": [
    "nim"
  ],
  "installDirs": [],
  "installFiles": [],
  "installExt": [],
  "requires": [
    {
      "name": "nim",
      "str": ">= 0.20.0",
      "ver": {
        "kind": "verEqLater",
        "ver": "0.20.0"
      }
    },
    {
      "name": "cligen",
      "str": ">= 0.9.38",
      "ver": {
        "kind": "verEqLater",
        "ver": "0.9.38"
      }
    },
    {
      "name": "foreach",
      "str": ">= 1.0.2",
      "ver": {
        "kind": "verEqLater",
        "ver": "1.0.2"
      }
    }
  ],
  "bin": [
    "golden"
  ],
  "binDir": "",
  "srcDir": "",
  "backend": "c",
  "ref": "refs/tags/1.0.1"
}